CHANGELOG.md

```diff
@@ -15,6 +15,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 - Add styling guide documentation for customizing SDK and UI component CSS.
 - Extend and enabled style customizing capabilities on the SDK components.
+- Add `MeetingSessionConfiguration` as a required parameter to `MeetingManager`'s `join` method.
```

Comment:
nit: Can we update the CHANGELOG to include the motivation behind the changes too (just a quick sentence)? :)

Reply:
Added

> With this change the builders have direct access to `MeetingSessionConfiguration`, this will allow more flexibility to customize the `MeetingSession`.